### PR TITLE
Edit class styling

### LINKED ISSUE #121: [Editor] Edit styling for edit modal - editor plan classes

### PR DESCRIPTION
**Pull request by Marcellia**
This closes #121

This issue includes
* the styling update for Edit Class Modal
* ability to add classwork type
* minor refactoring 

Pending discussion: Styling for Drop down/ Select list for Class types as Select component can be used for tags as well (looking into [react-select](https://www.npmjs.com/package/react-select) component that can be reused for tags as well )  

1EditClassModal

### CODE REVIEW

File: src/pages/Editor/Classes/UpdateClassModals/EditClassModal.tsx [hunk 1]
```diff
@@ -4,35 +4,75 @@
...
+  margin: 2% 5% 2% 5%;
+  background-color: transparent;
+  color: ${({ theme }) => theme.color.primary};
+  padding: 0;
+  border: none;
+  &:hover {
+    font-weight: bold;
+    cursor: pointer;
+    text-decoration: underline;
+  }
```
[the remark this answers is not in the document]
Comment (Marcellia) [reply src/pages/Editor/Classes/UpdateClassModals/EditClassModal.tsx:75]: How about LeftArrowButton? as the function can change as per use but it aptly describes the feature.

[the remark this answers is not in the document]
Comment (jpatrici) [reply src/pages/Editor/Classes/UpdateClassModals/EditClassModal.tsx]: hmm that's very specific to having a left arrow icon in the button. I was thinking more of like just a button that looks like a link, and you can extend that and modify it on your own.
```
export const LinkButton = styled(Button)` // Button here is the generic Button component from Button.tsx
   // the styling for generic Link button goes here
`;
```
and if you want to add a left arrow icon, you can do
```
<LinkButton>
   <LeftArrowIcon /> back to Syllabus
</LinkButton>

or if you want to style it even more:

export const StyledLinkButton = styled(LinkButton)`
   // some specific styling
`;

<StyledLinkButton>
   <LeftArrowIcon /> back to Syllabus
</StyledLinkButton>
```

This is how I do it for the `PlanCard` button
```
export const PlanCardButton = styled(PrimaryButton)`
  font-size: 0.9rem;
  width: fit-content;
`;

<PlanCardButton onClick={handleClick}>
    <ArrowIcon />
    Go to Syllabus
</PlanCardButton>
```

File: src/pages/Editor/Classes/styles.tsx [hunk 2]
```diff
@@ -14,3 +15,39 @@ export const StyledDateString = styled.span`
...
 `;
+
+export const StyledForm = styled.form`
+  margin: 0 5% 0 5%;
+  font-family: 'Poppins';
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+`;
```
Comment (jpatrici) [inline src/pages/Editor/Classes/styles.tsx:26]:
```suggestion
export const StyledForm = styled(Form)`
  margin: 0 5% 0 5%;
  gap: 1em;
`;
```
if possible, try to not use % for the margin

File: src/pages/Editor/Classes/styles.tsx [hunk 3]
```diff
@@ -14,3 +15,39 @@ export const StyledDateString = styled.span`
...
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 30px;
+    letter-spacing: 0.5px;
+  }
+`;
+export const AddNewClassWorkBtn = styled(PrimaryButton)`
+  border-radius: 20px;
+  position: absolute;
+  left: 7%;
```
Comment (jpatrici) [inline src/pages/Editor/Classes/styles.tsx:52]: is it possible to not use % here?

File: src/pages/Editor/Classes/styles.tsx [hunk 4]
```diff
@@ -14,3 +15,39 @@ export const StyledDateString = styled.span`
...
+  background: ${({ theme }) => theme.color.secondary};
+  display: flex;
+  justify-content: space-between;
+  margin: 2px 0;
+  padding: 10px;
+
+  span {
+    height: 30px;
+    font-weight: 600;
+    font-size: 16px;
```
Comment (jpatrici) [inline src/pages/Editor/Classes/styles.tsx:44]: should be rem instead

**Review by jpatrici — COMMENTED** [review]
Nice addition on the a11y 🙌 Added some more comments about the styling, any font size should be in `rem`, and padding/margin should be in `em`

File: src/components/Input.tsx [hunk 5]
```diff
@@ -21,16 +20,50 @@ export const InputField = styled.label`
...
 export const TextArea = styled.textarea<InputFieldProps>`
   background: none;
-  border: 1px solid ${({ theme }) => theme.color.black};
+  border: 1px solid ${({ theme }) => theme.color.grey};
   border-color: ${({ hasError, theme }) => hasError && theme.color.red};
   border-radius: 4px;
   min-height: 8rem;
   width: 11rem;
-  color: ${({ theme }) => theme.color.black};
+  color: ${({ theme }) => theme.color.darkGrey};
```
[the remark this answers is not in the document]
Comment (Marcellia) [reply src/components/Input.tsx:66]: I might have misunderstood,  I have now updated place holder to be dark grey & text as black.

**Pull request closed without merging**